src/scry/workspace.cr

```diff
@@ -27,7 +27,7 @@ module Scry
         @open_files[file.filename] = {file, Completion::MethodDB.new}
       else
         file_dependencies = @dependency_graph[file.filename].descendants.map &.value
-        method_db = Completion::MethodDB.generate(file_dependencies)
+        method_db = Completion::MethodDB.generate((file_dependencies + [file.filename]).uniq)
```

Comment:
Maybe do sth like:
```cr
file_dependencies << file.filename
method_db = Completion::MethodDB.generate(file_dependencies.uniq!)
```
To avoid 2 temporary arrays (one for the concatenation & one for the uniq call)